.github/workflows/test-download-artifact-s3-action.yml

```diff
@@ -0,0 +1,118 @@
+name: Test Download Artifact S3 Action
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/actions/download-artifact-s3/**'
+      - '!**/*.md'
+  pull_request:
+    paths:
+      - '.github/actions/download-artifact-s3/**'
+      - '!**/*.md'
+
+jobs:
+
+  build:
+    name: test-download-artifact-s3-build
+
+    strategy:
+      matrix:
+        runs-on: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
```suggestion
        runs-on: [linux.2xlarge, macos-m1-stable, windows.4xlarge.nonephemeral]
```

Reply:
This should make it so you don't need credentials

Reply:
Thank you ! I updated the code